server/views/components/personal-transactions/template.njk

```diff
@@ -29,107 +27,144 @@
         </a>
       </li>
     {% endfor %}
-    {% if (data.selected === 'damage-obligations') or (data.shouldShowDamageObligationsTab) %}
+    {% if (data.selected === 'damage-obligations') or data.shouldShowDamageObligationsTab %}
```

Comment:
was just wondering whether you had considered modelling this as 3 different routes rather than hiding/showing each section using javascript? 

imagine it would have simplified the service layer and pages and you could more easily track who was accessing each which tab. 

We do something similar in use of force, would there have been any negative side effects from doing it that way?

Reply:
Probably not, I think it's just evolved in to this as we've worked on it. 100% on splitting things out, I've just thrown a commit up that splits out the routes and templates to reduce the amount of conditional logic